### ISSUE TITLE
filter for exact value of terrain & difficulty

### ISSUE THREAD

**Issue by mapa6** [cgeo/cgeo#8570]
Why we need in filter terrain higher than 5 ?
And why we can not filter caches with exact terrain (for example only T1 instead of T1+1,5) ?
Thanks for check and improve.

**Comment by mapa6** [comment]
![IMG_20200703_010010](https://user-images.githubusercontent.com/43787170/86415889-aa029b00-bcc8-11ea-9666-34732898b664.jpg)

**Comment by mapa6** [comment]
The same options in difficulty filter - unable to filter exact D, only with combinations (D1+1,5 etc.)

**Comment by UniQP** [comment]
https://extremcaching.com/ has terrain ratings up to 7.

**Comment by Lineflyer** [comment]
@mapa6 
Is your question clarified with that, or would you like to rephrase this to have filters for each different T-value rather than the existing combined filters?

**Comment by mapa6** [comment]
Rephrase to have filters for each different T-value rather than the existing combined filters.

**Comment by Lineflyer** [comment]
thanks @mapa6  Will add the appropriate labels.
